debian/bullseye/foreman-release/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+include /usr/share/dpkg/pkg-info.mk
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+export RELEASE =  $(shell git branch --show-current)
```

Comment:
```suggestion
```

You don't build-depend on `git`, so it's not available, but we also don't need it, as we can use `$DEB_UPSTREAM_VERSION` (which is `9999` for nightly)